doc/man/cgset.1

```diff
@@ -19,6 +19,12 @@ Set the parameters of input cgroups.
 is the name of the cgroup which should be changed.
 This parameter can be used multiple times.
 
+.TP
+.B -b
+Ignores the default systemd delegated hierarchy path and
```

Comment:
Same capitalization question. Ironically, it looks like `cgdelete.1` capitalized every entry. Weird.